### DISCUSSION TITLE
Qucs-S 24.2.1

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#669] [Announcements]
This release contains no new features except the fixing of the critical bugs.

## General improvements

* Fixed zero output on DC bias simulation (F8) #667
* Fixed build failure of the Fedora RPM because of RPATH in qucsator_rf

<hr /><em>This discussion was created from the release <a href='https://github.com/ra3xdh/qucs_s/releases/tag/24.2.1'>Qucs-S 24.2.1</a>.</em>

**Comment by arktrin** [comment]
> AppImage for non-mentioned Linux distributions; use this download link below for file Qucs-S-x86_64-Linux.AppImage.zip
There is no Qucs-S-x86_64-Linux.AppImage.zip in files for this release.

**Comment by ra3xdh** [comment]
I will upload AppImage later. Please wait. What distribution are you using? I am providing the DEB/RPM for the most of distributions. Arch provides packages using AUR. AppImage is not a recommended way to install Qucs-S on Linux. It should be used only if no DEB/RPM is available.

**Comment by ra3xdh** [comment]
I have just uploaded AppImage on the release page.

**Comment by arktrin** [comment]
Thanks!
